daemons/execd/pacemaker-execd.c

```diff
@@ -487,8 +487,15 @@ main(int argc, char **argv, char **envp)
      */
     unsetenv("NOTIFY_SOCKET");
 
-    /* Used by RAs - Leave owned by root */
-    crm_build_path(CRM_RSCTMP_DIR, 0755);
+    {
+        // Temporary directory for resource agent use (leave owned by root)
+        int rc = pcmk__build_path(CRM_RSCTMP_DIR, 0755);
+
+        if (rc != pcmk_rc_ok) {
+            crm_warn("Could not create resource agent temporary directory "
+                     CRM_RSCTMP_DIR ": %s", pcmk_rc_str(rc));
+        }
```

Comment:
If this fails, it's okay for pacemaker-execd to continue?

Reply:
Yes, this is not used by Pacemaker itself, but by some resource agents. If it doesn't exist, they can provide error reporting/recovery as desired, without affecting the cluster itself or agents that don't use it.